File: Serene/Serene.Script/Meeting/Meeting/MeetingGrid.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System;
    using System.Collections.Generic;
    using System.Runtime.CompilerServices;

    [ColumnsKey("Meeting.Meeting"), IdProperty("MeetingId"), NameProperty("MeetingGuid"), IsActiveProperty("IsActive")]
    [DialogType(typeof(MeetingDialog)), LocalTextPrefix("Meeting.Meeting"), Service("Meeting/Meeting")]
    public class MeetingGrid : EntityGrid<MeetingRow>
    {
        public MeetingGrid(jQueryObject container)
            : base(container)
        {
        }
    }

    // Please remove this partial class or the first line below, after you run ScriptContexts.tt
    [Imported, Serializable, PreserveMemberCase] 
    public partial class MeetingRow
    {
    }
}

File: Serene/Serene.Web/Modules/Meeting/Meeting/MeetingGrid.ts

namespace Serene.Meeting {
    
    @Serenity.Decorators.registerClass()
    export class MeetingGrid extends Serenity.EntityGrid<MeetingRow, any> {
        protected getColumnsKey() { return 'Meeting.Meeting'; }
        protected getDialogType() { return MeetingDialog; }
        protected getIdProperty() { return MeetingRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingRow.localTextPrefix; }
        protected getService() { return MeetingService.baseUrl; }

        constructor(container: JQuery) {
            super(container);
        }
    }
}